projects/addon-mobile/components/pull-to-refresh/pull-to-refresh.service.ts

```diff
@@ -33,6 +33,7 @@ export class TuiPullToRefreshService extends Observable<number> {
         startWith(null),
         switchMap(() =>
             tuiTypedFromEvent(this.element, 'touchstart', {passive: true}).pipe(
+                filter(() => !this.el.nativeElement.querySelector('tui-dialog')),
```

Comment:
Let's move it to one `filter`, together with the next line. Also let's move `tui-dialog` to a constant and make it `tui-dialog, tui-dropdown, tui-dropdown-mobile` just to be sure.